tests/languages/swift-server/Tests.swift

```diff
@@ -68,36 +68,57 @@ class Tests: XCTestCase {
         let result = try await general.redirect()
         print((result as! [String: Any])["result"] as! String)
 
-        var url = URL(fileURLWithPath: "\(FileManager.default.currentDirectoryPath)/../../resources/file.png")
-        var buffer = ByteBuffer(data: try! Data(contentsOf: url))
-        var file = File(name: "file.png", buffer: buffer)
-        mock = try await general.upload(x: "string", y: 123, z: ["string in array"], file: file, onProgress: nil)
-        print(mock.result)
-
-		url = URL(fileURLWithPath: "\(FileManager.default.currentDirectoryPath)/../../resources/large_file.mp4")
-        buffer = ByteBuffer(data: try! Data(contentsOf: url))
-        file = File(name: "large_file.mp4", buffer: buffer)
-        mock = try await general.upload(x: "string", y: 123, z: ["string in array"], file: file, onProgress: nil)
-        print(mock.result)
-
         do {
-            try await general.error400()
+            var file = InputFile.fromPath("\(FileManager.default.currentDirectoryPath)/../../resources/file.png")
+            mock = try await general.upload(x: "string", y: 123, z: ["string in array"], file: file, onProgress: nil)
+            print(mock.result)
         } catch let error as AppwriteError {
             print(error.message)
         }
-
-        do {
-            try await general.error500()
-        } catch {
-            print(error.localizedDescription)
-        }
-
-        do {
-            try await general.error502()
-        } catch {
-            print(String(describing: error))
-        }
-
-        try! await general.empty()
+//         do {
```

Comment:
@abnegate All these tests are commented out?

Reply:
This was WIP at the time, sorted now 👍